test/test_s2i.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+import logging
+import os
+
+from conu import DockerBackend, S2IDockerImage
+
+import pytest
+
+image_name = os.environ.get("IMAGE_NAME", "ruby")
```

Comment:
The tests are failing when using default ruby image, on missing "io.openshift.s2i.scripts-url" label. We could push container-images/ruby image to docker.io/modularitycontainers/ruby-24 and use it in tests. It is also referenced in s2i build documentation - https://github.com/container-images/ruby/blob/master/README.md#usage.

Reply:
Or we could prioritize implementing building capability in conu and build the image from Dockerfile when it is not provided in IMAGE_NAME variable

Reply:
I will add a makefile so that the image is built first and then tested.